bot/cogs/defcon.py

```diff
@@ -146,6 +146,8 @@ def mod_log(self) -> ModLog:
             await ctx.send(self.build_defcon_msg(action, error))
             await self.send_defcon_log(action, ctx.author, error)
 
+            self.bot.stats.gauge("defcon.days", days)
```

Comment:
This is a good stat too but I meant to track the duration it is enabled, not the threshold used.

Reply:
I see, we don't have that information on demand after a restart though so this would involve changes to the defcon storage in the site KV store right?

Reply:
Yeah, I think so. It just stores JSON as the value IIRC. If you think it's worth it then let's go for it.

Reply:
Added